src/main/java/no/nav/pto/veilarbportefolje/kafka/KafkaConfigCommon.java

```diff
@@ -84,7 +77,7 @@ public enum Topic {
         VEILEDER_TILORDNET("pto.veileder-tilordnet-v1"),
         ENDRING_PAA_NY_FOR_VEILEDER("pto.endring-paa-ny-for-veileder-v1"),
         ENDRING_PA_MAL("pto.endring-paa-maal-v1"),
-        SIST_LEST("aapen-fo-veilederHarLestAktivitetsplanen-v1"),
+        SIST_LEST("pto.veileder-har-lest-aktivitetsplanen-v1"),
```

Comment:
Is this ready now? 🤔

Reply:
@Mathiamu  topic is ready, just waiting for dab to approve https://github.com/navikt/veilarblest/pull/70 and to start to produce on Aiven